TESTS.md

```diff
@@ -0,0 +1,38 @@
+### [<= Back to `README.md`](./README.md)
+## Running Unit Tests
+### Create `token.txt` in `~/Smart-Ledger/Back-End/tests/`
+```shell
+touch tests/token.txt
```

Comment:
I would also note that the token expires once in a while so it'll need to be updated in this file when that happens